web/src/components/ImageViewer/slick-theme.min.css

```diff
@@ -0,0 +1,139 @@
+@charset 'UTF-8';
```

Comment:
can we do some of these in tailwind?

Reply:
For react slick slider? Not really, not without rebuilding the image viewer from scratch. This package has a lot of customizable API options included and is quite easy to implement.

I'm sure we COULD re-do it in tailwind as a fully custom component from scratch however it will take some time. 

I'm very open to any other options - using react slick slider seemed like the most efficient and relatively lightweight option to me.